t/destructive/testers.go

```diff
@@ -0,0 +1,33 @@
+package destructive
```

Comment:
maybe call this file `utils.go`?

Reply:
i don't think that utils are appropriate name here, this are supposed to be a backends for introducing all kinds of errors